Utilities/PlayerUtils.cs

```diff
@@ -889,6 +889,23 @@ public static void HideAccessories(this Player player, bool hideHeadAccs = true,
         /// <returns>Returns <see langword="true"/> if the player CAN'T use the item.</returns>
         public static bool CantUseHoldout(this Player player, bool needsToHold = true) => player == null || !player.active || player.dead || (!player.channel && needsToHold) || player.CCed || player.noItems;
 
+        /// <summary>
+        /// A short method that heals the player.
+        /// All direct heals in Calamity should use this.
+        /// </summary>
+        /// <param name="player">The player being healed.</param>
+        /// <param name="amount">The amount of life being healed.</param>
+        /// <param name="healEffect">Whether the heal CombatText should be displayed.</param>
+        /// <param name="broadcast">Whether the heal CombatText should be synced.</param>
+        public static void HealPlayer(this Player player, int amount, bool healEffect = true, bool broadcast = true)
```

Comment:
Consideration:
- This does not sync the between player, and it depends on external `NetMessageID.SpiritHeal` call, Can we move this behaviour inside `HealPlayer`?

Reply:
SpiritHeal wasn't included in 90% of the cases, so i'd say it should probably be excluded.